src/tailwind.preset.js

```diff
@@ -1,32 +1,54 @@
+import plugin from 'tailwindcss/plugin.js';
+
 import colors from 'tailwindcss/colors.js';
 
 export default {
   theme: {
-    colors: {
-      transparent: 'transparent',
-      current: 'currentColor',
-      white: colors.white,
-      black: colors.black,
-      'grey-0': '#fafafa',
-      'grey-1': '#f2f2f2',
-      'grey-2': '#ececec',
-      'grey-3': '#dbdbdb',
-      'grey-4': '#a6a6a6',
-      'grey-5': '#9c9c9c',
-      'grey-6': '#737373',
-      'grey-7': '#595959',
-      'grey-8': '#3f3f3f',
-      'grey-9': '#202020',
-      brand: {
-        dark: '#84141e',
-        DEFAULT: '#bd1c2b',
+    extend: {
+      borderColor: {
+        DEFAULT: '#dbdbdb',
+      },
+      colors: {
+        transparent: 'transparent',
+        current: 'currentColor',
+        white: colors.white,
+        black: colors.black,
+        'grey-0': '#fafafa',
+        'grey-1': '#f2f2f2',
+        'grey-2': '#ececec',
+        'grey-3': '#dbdbdb',
+        'grey-4': '#a6a6a6',
+        'grey-5': '#9c9c9c',
+        'grey-6': '#737373',
+        'grey-7': '#595959',
+        'grey-8': '#3f3f3f',
+        'grey-9': '#202020',
+        green: {
+          success: '#00a36d',
```

Comment:
What is the effect of nesting here?

Reply:
I'll answer this in a specific way, but let me know if you were after an explanation of _why_ one would nest here.

Tailwind has a [nested object syntax](https://v1.tailwindcss.com/docs/customizing-colors#nested-object-syntax) it supports for colors. Paraphrasing the docs, the keys will "combine with parent keys" to form such classnames as `bg-green-success` or `bg-brand-dark`. Note that the special `DEFAULT` key is handled specially. Such that we end up with `bg-brand` and `bg-brand-dark` from this preset.

Does this help?

Reply:
Yes, thanks.